src/python/Python-2.7.8/setup.py

```diff
@@ -803,8 +803,8 @@ def detect_modules(self):
                                libraries=math_libs) )
         # Detect SSL support for the socket module (via _ssl)
         search_for_ssl_incs_in = [
-                              '/usr/local/ssl/include',
                               '/usr/sfw/include/',
+                              '/usr/local/ssl/include',
```

Comment:
Yes. 
If we put '/usr/local/ssl/include' first, the build fails with unsupported hash types errors. It loads the headers of the new OpenSSL version from that path (0.9.8zh), instead of the headers of the one that come with Solaris 10 and are on /usr/sfw/include/ (0.9.7) which we used on chevah_build script. But.. the libraries loaded will still be from /usr/sfw/lib (0.9.7) so it fails badly.

I just re-tested to confirm it fails if we change the path order there.